src/traces/parcoords/parcoords.js

```diff
@@ -569,18 +572,7 @@ module.exports = function(root, svg, styledData, layout, callbacks) {
                     .outerTickSize(2)
                     .ticks(wantedTickCount, d.tickFormat) // works for continuous scales only...
                     .tickValues(d.ordinal ? // and this works for ordinal scales
-                        sdom.map(function(dd, i) {
-                            if(texts) {
-                                var text = texts[i];
-                                if(text === null || text === undefined) {
-                                    return dd;
-                                } else {
-                                    return text;
-                                }
-                            } else {
-                                return dd;
-                            }
-                        }) :
+                        sdom.map(toText(Lib.identity), texts) :
```

Comment:
Thanks @alexcjohnson for the catch, thought it was magic that it still worked! Turns out, the `domainScale` change (same PR, higher up in the code) made `scale.domain()` to return the desired values anyway, which got mapped with the identity function (as `texts` was not supplied ie. falsey).